app.json

```diff
@@ -1,8 +1,13 @@
 {
     "env": {
-        "TOKEN": {
-            "description": "Discord bot token. This is from https://discord.com/developers/applications",
+        "MODMAIL_BOT_TOKEN": {
+            "description": "Discord bot token. Required to log in to discord.",
```

Comment:
I liked the older description better. Perhaps 
"Get this from ... url..."

Reply:
added extended descriptions to exports in bfe641c45f4645b1d8908086e29d436e2bcefd7b